src/assert.js

```diff
@@ -119,19 +119,17 @@ Assert.prototype = {
 	},
 
 	equal: function( actual, expected, message ) {
-		/*jshint eqeqeq:false */
 		this.pushResult( {
-			result: expected == actual,
+			result: expected == actual,		// eslint-disable-line eqeqeq
```

Comment:
No problem, I'm okay with the second option! I just wanted to be sure that's what you wanted. I'll implement that hopefully no later than tomorrow evening (US time).